### PR TITLE
Updated content with detailed steps through orientation

### PR DESCRIPTION
**Pull request by stevebrownlee**
This is a port of the orientation markdown files from the Python course. Since I've had two Python courses in a row, we've refined how orientation is done, and needs to be migrated back to C# course.

### CODE REVIEW

**Comment by stevebrownlee** [comment]
This branch is not ready for review yet

File: README.md [hunk 1]
```diff
@@ -46,15 +46,97 @@ While you are building out the Bangazon Platform, your leadership team will ensu
...
+## The Compiler Must Know the Type of Everything
 
+When your C# code is compiled into a executable program, the compiler must know the type of every variable before it will allow the code to be compiled. This is **very** different from JavaScript, in which a variable's type can change at any time.
 
+```js
+// Initial type of the variable `student` is string
+let student = "Bob"
 
+// In JavaScript, you're allowed to change the value to anything
+student = ["Bob", "Parson"]
```
Comment (JoeShep) [inline README.md]: Let's change "Bob" to "Fred"

File: README.md [hunk 2]
```diff
@@ -46,15 +46,97 @@ While you are building out the Bangazon Platform, your leadership team will ensu
...
+* integer
+* string
+* null
+* undefined
+* boolean
+* object
+
+You could combine them in all kinds of interesting ways and form very complex objects, or collections of objects, but in the end, everything was one of those types.
+
+In C# you can define your own custom variable types with a class. Here's a simple custom type that defines the properties and methods of a Tree object.
```
Comment (JoeShep) [inline README.md]: Typo: "Tree" is spelled "Treeeeeee"

**Comment by stevebrownlee** [comment]
I'll go ahead and merge this so it's available for @megducharme and @copypastedeveloper on Friday and Saturday respectively

**Pull request merged**